v4/export/sql.go

```diff
@@ -537,11 +512,11 @@ func createConnWithConsistency(ctx context.Context, db *sql.DB) (*sql.Conn, erro
 	return conn, nil
 }
 
-func buildSelectField(db *sql.Conn, dbName, tableName string, completeInsert bool) (string, error) {
+func buildSelectField(db *sql.Conn, dbName, tableName string, completeInsert bool) (string, int, error) {
```

Comment:
Could you please help add a comment before this line to explain the return values of this function?